tests/e2e/configs/scripts/update-custom-config.sh

```diff
@@ -48,6 +48,13 @@ sed -i -e "${line_number}s/enable = false/enable = true/g" $CHAIN_DIR/config/app
 line_number=$(get_next_line_number "Address defines the gRPC server address to bind to." $CHAIN_DIR/config/app.toml)
 sed -i -e "${line_number}s#address = \".*\"#address = \"0.0.0.0:9090\"#g" $CHAIN_DIR/config/app.toml
 
+if [ $METRICS ]; then
```

Comment:
this is really nice, we should make this part of the default scripts in the `charts/devnet/scripts` directly

Reply:
Absolutely, I missed that part in the PR.